include/deal.II/grid/manifold_lib.h

```diff
@@ -127,6 +116,131 @@ class SphericalManifold : public ChartManifold<dim, spacedim, spacedim>
 
 
 /**
+ * Manifold description for a spherical space coordinate system.
+ *
+ * You can use this Manifold object to describe any sphere, circle,
+ * hypersphere or hyperdisc in two or three dimensions, both as a co-dimension
+ * one manifold descriptor or as co-dimension zero manifold descriptor.
+ *
+ * The two template arguments match the meaning of the two template arguments
+ * in Triangulation<dim, spacedim>, however this Manifold can be used to
+ * describe both thin and thick objects, and the behavior is identical when
+ * dim <= spacedim, i.e., the functionality of SphericalManifold<2,3> is
+ * identical to SphericalManifold<3,3>.
+ *
+ * While PolarManifold reflects the usual notion of polar coordinates,
+ * it may not be suitable for domains that contain either the north
+ * pole, the south pole or the center.  Consider for instance the pair
+ * of points \f$x_1=(1,\pi/3,0)\f$ and \f$x_2=(1,\pi/3,\pi)\f$ in
+ * polar coordinates (laying on the surface of a sphere with radius
+ * one, on a parallel at height $\pi/3$. In this case connecting the
```

Comment:
btw, it should be "lying" not "laying". The former is passive (the point is there) whereas the latter is active (the point is being put there, as in "laying bricks")